### PR TITLE
History Remarks before Frozen Throne

### PR DESCRIPTION
**Pull request by zumbak04**
## Note before reading the changelog:
All the historical characters I added/mentioned should have Wowpedia links (**_W_** button in the upper right corner of the character tab).

### Azeroth changelog:
- **Stormwind nobles:**
  - Added **Ebonlocke** dynasty, rules of **Grand Hamlet** with 3 lore members in 603 : **Anson**, **Ello** and **Althea Ebonlocke**.
  - Added **Daris** dynasty, rulers of **Richgarden** with 2 lore members: **Farrin** and **Robin Daris**.
  - Added **Berrybuck** dynasty, rulers of **Addle** with 2 lore members: **Keira** and **Anduin Berrybuck**.
  - Added **Ference** dynasty, rulers of **Yorgen** with 2 lore members: **Crispin Ference** and his son, **Hogan Ference**.
  - Added **Dreuger** dynasty, rulers of **Raven Hill** with 2 lore members: **Rafe** and **Role Dreuger**.
  - Added **Millstipe** dynasty, rulers of **Great Mound** with 2 lore members: **Dorothea** and **Brodie Millstipe**. The entire dynasty except **Brodie Millstipe** secretly worship the **Forgotten Shadow**.
  - Added **Von'indi** dynasty, rulers of **Splinter Junction** with 2 lore members: **Catriona** and **Sirra Von'indi**.
  - Added **Lescovar** dynasty, rulers of **Goldshire**, with 2 lore members: **Gregor** and **Aldous Lescovar**.
  - Added **Erlgadin** dynasty, rulers of **Sentinel Hill**, with 2 lore members: **Egerton** and **Jerome Erlgadin**.
  - Added **Vanyst** dynasty, rulers of **Lonely Tower**, with 2 lore members: **Bolten** and **Morelan Vanyst**.
  - Added **Ridgewell** dynasty, rulers of **Lakeshire**, with 1 lore member: **Remington Ridgewell**.
  - Added **Wishock** dynasty, rulers of **Fulbrown**, with 1 lore member: **Baurles Wishock**.
- Ruler of **Ravel Hill** is feudal in the 603 bookmark.
- The Blasted Lands are ruled by 4 lore characters: **Daio**, **Grol Dreadmaul**, **Sevine** and **Allistarj**.
- No more worgens in Duskwood in the 603 bookmark.
- Added temples in **Splinter Junction** and **Misty Valley** to make them more developed.
- Added **Mai'zoth** that rules **Balia'mah** in the 603 bookmark. **Conrad Kurzen** is a loyal servant of **Mai'zoth**.
- Developed provinces of **Conrad Kurzen** in the 603 bookmark.
### Lordaeron changelog:
- **Hammerfall** was developed and is ruled by humans in the 583 bookmark. In the 603 bookmark, it's ruled by rebelled orcs.
- **Crestfall** is populated by Undead Void Orcs after the end of the Second War.
- Added **Duke Reginald Baradin II** that rules Tol Barad in the 603 bookmark.
### Kalimdor changelog:
- Developed goblin provinces in Tanaris.
- Developed the goblins provinces on Tel Abim, undeveloped the pygmy, hozen provinces.
- Saurfang's father name became a lore thing, so I used it.
### Script changelog (ignore if you don't know how scripting works):
- Moved all the historical global flags to `c_dark_portal.txt`, added comments.
- Made sure all the **Orc**, **Human**, **Tauren**, **Gnoll**, **Troll**, **Night Elf**, **High Elf**, **Goblin** historical characters have race trait added.
- Azerothian culture uses many its lore names like Varian, Edwin, Tiffin etc.
- Many human cultures uses many ancient Arathi lore names like Lordain, Anduin, Antonius etc.
### Northrend changelog:
- Gymer Banotil is no longer ruler and vassal of the Scourge in the 603 bookmark.

# How to test:
Check these changes in-game and tell your opinion.

### CODE REVIEW

File: localisation/000_Other_title_names.csv [hunk 1]
```diff
@@ -355,4 +355,5 @@ c_dazaralor_adj;Zuldazar;Zuldazar;Zuldazar;;Zuldazar;;;;;;;;;x
 d_zuldazar;Kings Bay;;;;;;;;;;;;;x
 d_zuldazar_adj;Kings Bay;;;;;;;;;;;;;x
 d_defias_brotherhood;Defias Brotherhood;;;;;;;;;;;;;x
-d_defias_brotherhood_adj;Defias;;;;;;;;;;;;;x
\ No newline at end of file
+d_defias_brotherhood_adj;Defias;;;;;;;;;;;;;xd_hammerfall;Hammerfall;;;;;;;;;;;;;x
```
Comment (MemeWizard) [inline localisation/000_Other_title_names.csv]: Merge went wrong?

Comment (zumbak04) [reply localisation/000_Other_title_names.csv]: Fixed in https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/pull/1063/commits/1ca8e6e7fb9cf4fd7f2333b996d9274cd1013de5.

File: history/characters/44000_baradin.txt [hunk 2]
```diff
@@ -152,44 +146,46 @@
...
 44015={
-	name=Lothar
+	name=Reginald
 	dynasty=22001
 	dna="pebmiifdpco"
 	culture=baradin religion=holy_light
 	martial=7 diplomacy=8 stewardship=4 intrigue=8 learning=8
-	add_trait=martial_cleric add_trait=temperate add_trait=arbitrary add_trait=shy add_trait = creature_human
+	add_trait=skilled_tactician add_trait=temperate add_trait=just add_trait=shy add_trait=brave
 	father=44014	#Gerold
```
[the remark this answers is not in the document]
Comment (zumbak04) [reply history/characters/44000_baradin.txt]: Fixed in https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/commit/90cc5ea60c3dcda351d78fbb5b724592352f0b39.

File: history/characters/53000_hillsbrad.txt [hunk 3]
```diff
@@ -5,9 +5,9 @@
...
 	culture=hillsbrad religion=holy_light
 	martial=7 diplomacy=5 stewardship=6 intrigue=8 learning=4
-	add_trait=tough_soldier add_trait=cynical add_trait=proud add_trait=lustful add_trait=deceitful add_trait = creature_human
+	add_trait=tough_soldier add_trait=cynical add_trait=proud add_trait=lustful add_trait=deceitful
 	disallow_random_traits = yes
-	533.9.1={ birth=yes }
+	533.9.1={ birth=yes add_trait=creature_human }
 	549.9.1={
 		# add_trait=class_warrior
 		# effect = { add_character_modifier = { name = hero_power_1 duration = -1 } }
```
[the remark this answers is not in the document]
Comment (zumbak04) [reply history/characters/53000_hillsbrad.txt]: Indeed.

File: history/provinces/2939 - Noonshade.txt [hunk 4]
```diff
@@ -4,11 +4,11 @@
...
 
 # Settlements
-max_settlements = 4
+max_settlements = 3
 
 b_noonshade = castle
-#b_makavru = city
-#b_risesh_2 = temple
+b_makavru = city
+# b_risesh_2 = temple
```
[the remark this answers is not in the document]
Comment (zumbak04) [reply history/provinces/2939 - Noonshade.txt]: Holdings got reshuffled.

File: history/titles/c_three_corners.txt [hunk 5]
```diff
@@ -23,9 +23,9 @@
 499.09.23={holder=261} #Anson[3]
 554.04.18={holder=263} #Brentan[3]
 557.12.03={holder=267} #Augustus[3]
+# Conquired by patron
```
[the remark this answers is not in the document]
Comment (zumbak04) [reply history/titles/c_three_corners.txt]: Never used Usurped in comments, but ok.

[the remark this answers is not in the document]
Comment (zumbak04) [reply history/titles/c_three_corners.txt]: Changed in https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/commit/7b4c96eb2a258c8c264c85c7a6a3bcb9b52f7de8.

File: history/provinces/44 - Raven Hill.txt [hunk 6]
```diff
@@ -4,13 +4,13 @@
...
-max_settlements = 4
+max_settlements = 5
 
+b_bago = castle
 b_raven_hill = city
 b_raven_hill_2 = temple
 #b_nerton = temple
-#b_bago = castle
-#b_hildom = city
+b_hildom = city
```
[the remark this answers is not in the document]
Comment (zumbak04) [reply history/provinces/44 - Raven Hill.txt]: Changed in https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/commit/13cb8a22a3c48667f2c9a8b91b3eb091f489fc7b.

File: history/characters/1_azerothian.txt [hunk 7]
```diff
@@ -9029,118 +8813,5136 @@
...
+	culture=azerothian religion=holy_light
+	martial=5 diplomacy=8 stewardship=5 intrigue=6 learning=4
+	add_trait=mastermind_theologian add_trait=trusting add_trait=just add_trait=cruel add_trait=lustful
+	add_trait=brave add_trait=cynical
+	father=1281	#Ripley
+	650.6.5={ birth=yes add_trait=creature_human }
+	735.4.8={ death=yes }
+}
+
+# dynasty=12 Millstipe Nobles of Darkshire
```
Comment (MemeWizard) [inline history/characters/1_azerothian.txt]: It is ok that, like, all of them are forsaken?

**Comment by zumbak04** [comment]
> > [characterhistory.cpp:798]: Trying to add trait 'Cruel' despite character '1152' already having the opposite of it.
> > [characterhistory.cpp:798]: Trying to add trait 'Kind' despite character '1247' already having the opposite of it.
> > [characterhistory.cpp:798]: Trying to add trait 'Kind' despite character '1254' already having the opposite of it.
> > [characterhistory.cpp:798]: Trying to add trait 'Cruel' despite character '1280' already having the opposite of it.
> > [characterhistory.cpp:798]: Trying to add trait 'Cruel' despite character '1308' already having the opposite of it.
> > [characterhistory.cpp:798]: Trying to add trait 'Cruel' despite character '1420' already having the opposite of it.
> > [characterhistory.cpp:798]: Trying to add trait 'Kind' despite character '1553' already having the opposite of it.
> > [characterhistory.cpp:798]: Trying to add trait 'Kind' despite character '1591' already having the opposite of it.

Fixed in https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/commit/7d194b91a8af482fe9559a790bedf8adacabb959.

File: history/characters/1_azerothian.txt (hunk 7 above)
Comment (zumbak04) [reply history/characters/1_azerothian.txt]: Yes. Mentioned it in the changelog.

File: history/titles/c_misty_valley.txt [hunk 8]
```diff
@@ -1,23 +1,40 @@
...
+# 419.05.20={holder=1456} #Nico[13]
+# 470.08.17={holder=1461} #Chapman[13]
+# 499.11.09={holder=1467} #Fitch[13]
+500.1.1={holder=1467} #Fitch[13]
 500.1.1={
 	liege=k_stormwind
-	law = cognatic_succession
-	law = succ_primogeniture
 	holder=864 #Alcott[9]
 }
```
[the remark this answers is not in the document]
Comment (zumbak04) [reply history/titles/c_misty_valley.txt]: Fixed in https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/commit/fae8402d378fe78423c4b16b301c4a61b3bfa7b0.

**Pull request merged**